packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx

```diff
@@ -40,6 +41,7 @@ const messages = {
   done: 'Done',
   invalidRemixUrl: 'Please paste a valid Audius track URL',
   missingRemixUrl: 'Must include a link to the original track',
+  remixAccessError: 'Must have access to parent track to remix',
```

Comment:
This copy is not final. No official copy has been provided for this scenario yet. This should be brought up in the next QA session to make sure it's fine with product/design

Reply:
Let's just say 'Must have access to the original track'